exchanges/kucoin/kucoin_test.go

```diff
@@ -1094,11 +1094,45 @@ func TestGetBasicFee(t *testing.T) {
 
 func TestGetTradingFee(t *testing.T) {
 	t.Parallel()
+
+	_, err := ku.GetTradingFee(context.Background(), nil)
+	if !errors.Is(err, currency.ErrCurrencyPairsEmpty) {
+		t.Fatalf("received %v, expected %v", err, currency.ErrCurrencyPairsEmpty)
+	}
+
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku)
 
-	_, err := ku.GetTradingFee(context.Background(), "BTC-USDT")
+	avail, err := ku.GetAvailablePairs(asset.Spot)
 	if err != nil {
-		t.Error("GetTradingFee() error", err)
+		t.Fatal(err)
+	}
+
+	pairs := currency.Pairs{avail[0]}
+	btcusdTradingFee, err := ku.GetTradingFee(context.Background(), pairs)
+	if !errors.Is(err, nil) {
+		t.Fatalf("received %v, expected %v", err, nil)
+	}
+
+	if len(btcusdTradingFee) != 1 {
+		t.Error("GetTradingFee() error, expected 1 pair")
+	}
+
+	// NOTE: Test below will error out from an external call as this will exceed
+	// the allowed pairs. If this does not error then this endpoint will allow
+	// more items to be requested.
+	pairs = append(pairs, avail[1:11]...)
+	_, err = ku.GetTradingFee(context.Background(), pairs)
+	if errors.Is(err, nil) {
+		t.Fatalf("received %v, expected %v", err, "code: 200000 message: symbols size invalid.")
```

Comment:
I think you will be annoyed that if Kucoin ever changes their policy, this test will fail until this is updated. I'm assuming you are placing this here as a canary to let you know to investigate the API change. I don't think its worth testing, but I'm also fine with it staying for you.